backend/app/api/routes/conversations.py

```
@@ -0,0 +1,487 @@
+    current_user: CurrentUser,
+    conversation_id: uuid.UUID,
+    message_request: AddMessageRequest,
+) -> AIConversationPublic:
+    """向对话添加消息并获取AI响应。"""
+    
+    # 获取对话
+    conversation = session.exec(
+        select(AIConversation).where(
+            AIConversation.id == conversation_id,
+            AIConversation.user_id == current_user.id,
+            AIConversation.is_active == True,
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if AIConversation.is_active:` for truth checks

Replace with `AIConversation.is_active`

(E712)